File: TEST data for large repo/REPO_Copy-6/Blueprism/BluePrism.Api/BluePrism.Api.Models/Dashboard/ResourceUtilization.cs
namespace BluePrism.Api.Models.Dashboard
{
    using System;
    using System.Collections.Generic;

    public class ResourceUtilization
    {
        public Guid ResourceId { get; set; }
        public string DigitalWorkerName { get; set; }
        public DateTimeOffset UtilizationDate { get; set; }
        public IEnumerable<int> Usages { get; set; }
    }
}

File: TEST data for large repo/repo-Copy5/Blueprism/BluePrism.Server.Domain.Models/Dashboard/ResourceUtilization.vb
Imports System.Runtime.Serialization

Namespace Dashboard
    <Serializable>
    <DataContract([Namespace]:="bp")>
    Public Class ResourceUtilization
        <IgnoreDataMember>
        Public ResourceId As Guid
        <IgnoreDataMember>
        Public DigitalWorkerName As String
        <IgnoreDataMember>
        Public UtilizationDate As DateTimeOffset
        <IgnoreDataMember>
        Public Usages As IEnumerable(Of Integer)
    End Class
End NameSpace
